src/components/Lead/Lead.module.css

```diff
@@ -97,6 +97,10 @@
         margin-right: auto;
         margin-left: 24px;
     }
+
+    .caption {
```

Comment:
this change does not relevant to the grid section and should be moved to another PR, but you can leave it here if you do not want to create another one. Personally, I do not like it, but we did not have this requirement in the beginning of the project.